src/Serilog.Sinks.AzureDataExplorer/AzureDataExplorerSinkOptions.cs

```diff
@@ -78,30 +120,43 @@ public int QueueSizeLimit
         public string ApplicationKey { get; private set; }
         public X509Certificate2 ApplicationCertificate { get; private set; }
         public string Authority { get; private set; }
-        public bool SendX5c { get; private set; }
+        public bool SendX5C { get; private set; }
         public string AzureRegion { get; private set; }
         public Azure.Core.TokenCredential TokenCredential { get; private set; }
 
         public AzureDataExplorerSinkOptions()
```

Comment:
We can add tests for MSI , if we have great! else you can look at the article below

[$env:AZURE_TENANT_ID=""
$env:AZURE_CLIENT_ID=""
$env:AZURE_CLIENT_SECRET="". ](https://www.rahulpnath.com/blog/azure-managed-service-identity-and-local-development/)